tools/mmp/driver.cs

```diff
@@ -824,7 +855,25 @@ static void AOTBundle ()
 
 			Func <FileInfo, bool> aotableFile = x => {
 				string extension = x.Extension.ToLowerInvariant ();
-				return extension == ".exe" || extension == ".dll";
+				if (extension != ".exe" && extension != ".dll")
+					return false;
+
+				Console.WriteLine (x.Name);
+				if (aot_ignore.Contains (x.Name))
+					return false; 
+				if (aot_ignore.Count > 0)
+					Console.WriteLine (aot_ignore[0]);
+
+				switch (aot) {
+				case AotType.All:
+					return true;
+				case AotType.Platform:
+					return x.Name == "Xamarin.Mac.dll" || x.Name == "System.dll" || x.Name == "mscorlib.dll";
+				case AotType.Explicit:
+					return aot_explicit_reference.Contains (x.Name);
+				default:
+					throw new InvalidOperationException ($"AOTBundle with aot: {aot}?");
```

Comment:
This should have a better error number, maybe something like this: https://github.com/rolfbjarne/xamarin-macios/blob/framework-sdk/tools/mtouch/error.cs#L108